flow/record/adapter/elastic.py

```diff
@@ -25,6 +25,7 @@
 
 Optional arguments:
   [API_KEY]: base64 encoded api key to authenticate with (default: False)
+  [QUEUE_SIZE]: integer that sets the limit on the maximum number of records in the queue. Limits memory usage (default: 100000)
```

Comment:
Made the description a bit more concise:
```suggestion
  [QUEUE_SIZE]: maximum queue size for writing records; limits memory usage (default: 100000)
```